### PR TITLE
Add new NVA forms to lead form widget

### PR DESCRIPTION
**Pull request by quady**
To go along with this pull request: https://github.com/g5search/g5-client-leads-service/pull/203

### CODE REVIEW

**Comment by thescubageek** [comment]
@quady When this is ready (passed QA + code review), make a PR against 2.1.7-merge and update this PT so it gets merged in: https://www.pivotaltracker.com/story/show/102124406

**Comment by jcstringer** [comment]
:ship:

File: public/static/components/widgets/lead-form/edit.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   {{ widget.form_note | edit_text_field: "Form Note" }}
 
   <!-- Form Type -->
-  {% assign form_opts = '{"apply":"Apply", "contact":"Contact", "corp_contact":"Corp Contact", "maintenance":"Maintenance", "request":"Request", "suggestion":"Suggestion", "tour": "Tour", "inquiry":"Integrated Inquiry", "contact_us":"Integrated Contact Us", "online_application":"Integrated Online Application", "tour_request": "Integrated Tour Request"}' %}
+  {% assign form_opts = '{"apply":"Apply", "contact":"Contact", "corp_contact":"Corp Contact", "maintenance":"Maintenance", "request":"Request", "suggestion":"Suggestion", "tour": "Tour", "inquiry":"Integrated Inquiry", "contact_us":"Integrated Contact Us", "online_application":"Integrated Online Application", "tour_request": "Integrated Tour Request", "nva_contact": "NVA Contact Form", "nva_appt": "NVA Appointment Request"}' %}
```
Comment (thescubageek) [inline public/static/components/widgets/lead-form/edit.html]: Ultimately we should make this pull from an endpoint on the CLS so it doesn't have to manually updated each time we add a new form...

**Comment by thescubageek** [comment]
@quady Merged into 2.1.7-merge: https://github.com/g5search/g5-content-management-system/pull/843

**Pull request closed without merging**